blockware/pixel-dust/src/main.cpp

```diff
@@ -262,7 +265,10 @@ void setup(void) {
 
   if(!sand.begin()) err(1000); // Slow blink = malloc error
 
-  if(!accel.begin(Wire, ACCEL_ADDR)) err(250);  // Fast bink = I2C error
+  acc = new LIS2DW12Sensor(&Wire, 0x31U);
+  begin_retval = acc->begin();
+  Serial.printf("begin returned: %d\n", begin_retval);
```

Comment:
Two things:
* We haven't started serial yet at this point `Serial.begin`
* Use `DLOG` since it can also work for debugging over telnet